package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": {
```

Comment:
Dependencies should be at the bottom. Instead of writing them out yourself, you can just install them using the npm cli and they will be appended automatically at the bottom